javascript/javascript3/week1/script.js

```diff
@@ -0,0 +1,109 @@
+const userInput = document.getElementById("userInput");
+const submitButton = document.getElementById("submitButton");
+const displaySection = document.getElementById("display")
+
+submitButton.addEventListener("click", searchByCity)
+function searchByCity() {
+    const cityName = userInput.value;
+    if (cityName === "") {
+        alert("enter city name");
```

Comment:
When the page loads, the cityName will always be empty. 
Calling an alert function here will always break the UI/UX flow because the alert will always pop up every time the page loads and the city is empty. Not ideal from a user perspective.  
It would be better to load the page and if the user try to search without typing a city name, then you can display a div conditionally, alerting the user that it needs a city to get the weather.